admin/app/views/admin_main.scala.html

```diff
@@ -56,7 +56,7 @@
 
     </head>
     <body>
-        @admin_head(isAuthed)
+        @admin_head()
```

Comment:
does this still need to be a method being called? I don’t know much about Play 🤷

Reply:
probably not, but I didn't want to tinker too much